bin/varnishd/cache/cache_deliver_proc.c

```diff
@@ -49,30 +49,56 @@
  * r > 0:  Stop, breaks out early without error condition
  */
 int
```

Comment:
not just here, but in most of the touched functions: shouldn't the return value be changed into `enum vdp_status` also?